fuel-tx/src/transaction/repr.rs

```diff
@@ -33,3 +38,19 @@ impl From<&Transaction> for TransactionRepr {
         }
     }
 }
+
+impl TryFrom<Word> for TransactionRepr {
```

Comment:
I don't see where do we use it. Could you point please?=)

Reply:
It's used here : https://github.com/FuelLabs/fuel-vm/blob/4618cf67567d312a74d17328f72940efaf217e18/fuel-vm/src/interpreter/metadata.rs#L648
Which is a internal type that allow more clean matching for tx specific GTF here : https://github.com/FuelLabs/fuel-vm/blob/4618cf67567d312a74d17328f72940efaf217e18/fuel-vm/src/interpreter/metadata.rs#L540

Reply:
Okay, I see. I think we don't need to put unreachable use cases(`InvalidTransactionType`) into a panic reason. I think we should use `Bug` in the case if `Mint` transaction presents in VM, but it is not possible right now, because it doesn't implement `ExecutableTransaction` trait .

Reply:
I'm not sure I understand, I do nothing for now and open an issue ?

Reply:
Removed in https://github.com/FuelLabs/fuel-vm/pull/882/commits/a01e2a9b4d008065cf2010e2b294414cb83ac109

Reply:
Okey ty